js/api.js

```diff
@@ -1,41 +1,41 @@
 "use strict";
 
-let server = "https://fullstack.della-mura.com.br/";
-let subscriptionEndPoint = server + "api/subscription";
+const server = "https://fullstack.della-mura.com.br/";
+const subscriptionEndPoint = server + "api/subscription";
 
-function doPost(url, object, fnOk, fnError){
+function doPost(url, object, fnOk, fnError) {
     fetch(url, {
         method: 'post',
+        mode: "same-origin",
```

Comment:
Em https://developer.mozilla.org/en-US/docs/Web/API/Request/mode 
The associated mode, available values of which are:
(...)
    same-origin — If a request is made to another origin with this mode set, the result is simply an error. You could use this to ensure that a request is always being made to your origin.

Neste caso, a requisição está sendo feita para uma api em outro domínio, isso não causaria problemas?

Reply:
Correto, causaria sim.
 Foi erro meu nessa parte.